modules/ocf_desktop/manifests/defaults.pp

```diff
@@ -4,14 +4,12 @@
     source  => 'puppet:///modules/ocf_desktop/xsession/mimeapps.list';
   }
 
+  # temporary only, to get rid of files!
   file { '/usr/local/bin/pdf-open':
-      mode    => '0755',
-      source  => 'puppet:///modules/ocf_desktop/xsession/pdf-open',
-      require => Package['libimage-exiftool-perl'];
```

Comment:
is this package still needed on desktops? If not, maybe we can remove it from `packages.pp`

Reply:
Should be possible to remove this (it was added along with the pdf-open script in https://github.com/ocf/puppet/commit/e34981f26c4d9da12b367e74654ffe1736e7ed6d#diff-aec440fa3888a0f9bae5001579017f62), I agree it would be a good idea to do so.

Reply:
Will make a commit to do so, thanks!